src/components/doctor/patients/modals/CreateResolution.tsx

```diff
@@ -0,0 +1,48 @@
+import { useRef } from 'react';
+import Modal from '../../../UI/Modal';
+import {
+  StyledArea, StyledBtn, StyledControllers, StyledName, StyledText, StyledTitle, StyledWrapper,
+} from './ResolutionModal.styled';
+import useActions from '../../../../hooks/useActions';
+
+interface DropDownProps {
+  onClose: () => void,
+  text: string,
+  firstName: string,
+  lastName: string,
+  id: string
+}
+
+const CreateResolution = ({
+  onClose, text, firstName, lastName, id,
+}: DropDownProps) => {
+  const { createResolutionHandler } = useActions();
+  const textRef = useRef<HTMLTextAreaElement>(null);
+  const onCreateHandler = () => {
+    const resolutionData = {
+      resolution: textRef.current!.value,
+      appointmentID: id,
+    };
+    createResolutionHandler(resolutionData);
+  };
+  return (
+    <Modal onClose={onClose}>
+      <StyledWrapper>
+        <StyledTitle>Create a Resolution</StyledTitle>
+        <StyledName>
+          {firstName}
+          {' '}
```

Comment:
what is this for? Can we avoid it?

Reply:
this is for space firstname and lastname. Airbnb rules don't allow me to have 1 more expression per line. If it is not ok i could create an exception. What do you think?

Reply:
Okay, let it be.